Utilities/VisualStudioProjectTemplates/EdFi.ProjectTemplates.Extensions/EdFi.ProjectTemplates.Extensions.csproj

```diff
@@ -86,6 +86,7 @@
     <None Include="Artifacts\PgSql\Data\EdFi\ThisFileMustExist.txt" />
     <None Include="Artifacts\PgSql\Structure\EdFi\ThisFileMustExist.txt" />
     <None Include="Artifacts\Schemas\ThisFileMustExist.txt" />
+    <None Include="EdFi.Ods.Extensions.ExtensionName.nuspec" />
```

Comment:
This includes it in the template project but not when using the template to create a new extension.

Reply:
I changed now . Thanks